app/src/main/java/com/iic/shopingo/utils/StreamToString.java

```diff
@@ -0,0 +1,24 @@
+package com.iic.shopingo.utils;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+/**
+ * Created by asafg on 12/03/15.
+ */
+public class StreamToString {
+  public static String toString(InputStream stream) {
+    BufferedReader r = new BufferedReader(new InputStreamReader(stream));
+    StringBuilder total = new StringBuilder();
```

Comment:
You can use Guava CharStream to convert a stream to a string, see this code: https://github.com/iic-ninjas/lunchtime-android/blob/master/app/src/main/java/com/iic/lunchtime/dal/MigrationEngine.java#L38.

Also you need to close the reader in a finally close.